### PR TITLE
Login admin on create-admin

### LINKED ISSUE #169: Creating admin should directly print an access token as well
**User story**
When bootstrapping the first admin user an access token should be issued at the same time. This allows bootstrapping users even in modes where ES runs in memory and "create-admin" cannot modify the database.

**Acceptance Criteria**
- create admin also prints an access token

**Scope**
create admin

### PR DESCRIPTION
**Pull request by iead-cc**
### Description
When ES is run with create-admin, the admin is as well immediately logged in and a valid access token is generated

### Parent Issue
Closes #169 

### Comments for the Reviewers
- (What part of the diff is a good starting point)
- (How to test these changes)
- (Areas where extra attention/feedback is appreciated)

### Type(s) of Change (Split the PR if you check many)
- [x] Added user feature
- [ ] Added technical feature
- [ ] Added tests
- [ ] Refactor
- [ ] Bugfix
- [ ] Updated documentation
- [ ] Infrastructure changes (Terraform, GCP, K8s, other)
- [ ] CI/CD workflow changes

### Testing Checklist
- [ ] I have added/updated unit tests for functional changes.
- [ ] I have added/updated integration tests for changes that affect dependent systems.
- [ ] I have added/updated end-to-end tests for feature changes.

### General Checklist
- [x] I have pulled in the latest changes from master.
- [x] I have run `make lint`.
- [x] I have successfully run `make docker-up; make tests`.
- [ ] I have updated relevant workflows and deployment tools.
- [ ] I have updated/added relevant documentation (readme, doc comments, etc).
- [ ] I have added the license to new source code files.
- [x] I have spent some time looking over the full diff before creating this PR.

### CODE REVIEW

File: encryption-service/impl/authn/authn.go [hunk 1]
```diff
@@ -146,32 +146,56 @@ func (ua *UserAuthenticator) NewAdminUser(authStore interfaces.AuthStoreInterfac
 	requestID, err := uuid.NewV4()
 	if err != nil {
 		log.Fatal(ctx, err, "Could not generate uuid")
+		return err
```
Comment (bruno-cc) [inline encryption-service/impl/authn/authn.go:149]: I am not sure about this return after a fatal

File: encryption-service/impl/authn/authn.go [hunk 2]
```diff
@@ -146,32 +146,56 @@ func (ua *UserAuthenticator) NewAdminUser(authStore interfaces.AuthStoreInterfac
...
 
-	ctx = context.WithValue(ctx, contextkeys.AuthStorageTxCtxKey, authStoreTx)
+	ctxCreate := context.WithValue(ctx, contextkeys.AuthStorageTxCtxKey, authStoreTxCreate)
 	adminScope := users.ScopeUserManagement
-	userID, password, err := ua.NewUser(ctx, adminScope)
+	userID, password, err := ua.NewUser(ctxCreate, adminScope)
 	if err != nil {
-		log.Fatal(ctx, err, "Create user failed")
+		log.Fatal(ctxCreate, err, "Create user failed")
+		return err
```
Comment (bruno-cc) [inline encryption-service/impl/authn/authn.go:170]: same here

**Comment by bruno-cc** [comment]
@iead-cc see comments

File: encryption-service/impl/authn/authn.go (hunk 2 above)
Comment (iead-cc) [reply encryption-service/impl/authn/authn.go:170]: i was not sure what happens there... If the function just continues on that would be even more fatal i thought...

**Review by bruno-cc — APPROVED** [review]
Thanks!

**Pull request merged**